code/modules/jobs/job_types/assistant.dm

```diff
@@ -17,13 +17,22 @@ Assistant
 	paycheck = PAYCHECK_ASSISTANT // Get a job. Job reassignment changes your paycheck now. Get over it.
 	paycheck_department = ACCOUNT_CIV
 	display_order = JOB_DISPLAY_ORDER_ASSISTANT
+	var/new_player_title = "Intern"
 
 /datum/job/assistant/get_access()
 	if(CONFIG_GET(flag/assistants_have_maint_access) || !CONFIG_GET(flag/jobs_have_minimal_access)) //Config has assistant maint access set
 		. = ..()
 		. |= list(ACCESS_MAINT_TUNNELS)
 	else
 		return ..()
+/datum/job/assistant/get_title(mob/H)
+	var/final = title
+	var/client/the_client = H.client
+	if(the_client)
+		var/minutes = the_client.get_exp_living(TRUE)
+		if(minutes < (10 HOURS))
+			final = new_player_title
+	return final

```

Comment:
```suggestion

/datum/job/assistant/get_title(mob/H)
	var/final = title
	var/client/the_client = H.client
	if(the_client)
		var/minutes = the_client.get_exp_living(TRUE)
		if(minutes < 10 HOURS)
			final = new_player_title
	return final

```

Reply:
bold of you to assume byond will parse correctly.

Reply:
it will I promise 🤞 
also I don't like the lack of whitespace between the function and the one above it if you're set on being paranoid at least keep that pls